# HG changeset patch
# User Rik <rik@octave.org>
# Date 1343162426 25200
#      Tue Jul 24 13:40:26 2012 -0700
# Node ID f7bcfb3b96e701d057a567e82bdcab7bc38b06a2
# Parent  6239f5806c2699a11172e8352887b504d328bec0
doc: Fix typo in docstring for gco.m

* gco.m: Change 'and empty' to 'an empty'.

diff --git a/scripts/plot/gco.m b/scripts/plot/gco.m
--- a/scripts/plot/gco.m
+++ b/scripts/plot/gco.m
@@ -22,17 +22,17 @@
 ## Return a handle to the current object of the current figure, or a handle
 ## to the current object of the figure with handle @var{fig}.  The current
 ## object of a figure is the object that was last clicked on.  It is stored
 ## in the CurrentObject property of the target figure.
 ##
 ## If the last mouse click didn't occur on any child object of the figure,
 ## the current object is the figure itself.
 ##
-## If no mouse click occured in the target figure, this function returns and
+## If no mouse click occured in the target figure, this function returns an
 ## empty matrix.
 ##
 ## Note that the value returned by this function is not necessarily the same
 ## as the one returned by gcbo during callback execution.  An executing
 ## callback can be interrupted by another callback and the current object
 ## can be modified.
 ##
 ##@seealso{gcbo, gcf}
